static/js/composebox_typeahead.js

```diff
@@ -455,7 +455,11 @@ exports.content_typeahead_selected = function (item) {
             beginning += '@*' + item.name + '* ';
             $(document).trigger('usermention_completed.zulip', {user_group: item});
         } else {
-            beginning += '@**' + item.full_name + '** ';
+            beginning += '@**' + item.full_name;
```

Comment:
Where are we checking server version as said by @aero31aero, am I missing out something? 🤔 
https://github.com/zulip/zulip-mobile/issues/3000#issuecomment-447307063